staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation_test.go

```diff
@@ -660,6 +717,7 @@ func TestCelCompilation(t *testing.T) {
 
 			for i, expectedResult := range tt.expectedResults {
 				if !expectedResult.matches(compilationResults[i]) {
+					t.Logf(compilationResults[i].Error.Error())
```

Comment:
Handle error checking here properly with `t.Errorf` or `t.Fatalf`?

Reply:
This line was a little fragile + hard to generalize with the addition of `MessageExpressionError`, so I've removed it.